Gems/EMotionFX/Code/MCore/Source/Command.cpp

```diff
@@ -28,7 +28,7 @@ namespace MCore
 
 
     // constructor
-    Command::Command(const char* commandName, Command* originalCommand)
+    Command::Command(const AZStd::string& commandName, Command* originalCommand)
```

Comment:
It is more efficient to accept this parameter by value and then move the `comamndName` parameter into the `mCommandName`.
It prevents a second copy from being created when a literal is supplied

Reply:
Fixed.